components/brave_wallet/browser/eth_tx_state_manager.cc

```diff
@@ -36,7 +36,7 @@ EthTxStateManager::EthTxStateManager(
   rpc_controller_.set_disconnect_handler(base::BindOnce(
```

Comment:
@darkdh please check changes in this file, users can add custom networks and I changed it to work with custom networks as well.